org_status/org_status.py

```diff
@@ -84,6 +84,13 @@ def main():
     for Host, org in generate_fetch_jobs(args.orgs):
         token = None
 
+        try:
+            if not Host.get_status():
+                print(Host.HostName + ' is currently down')
```

Comment:
Please move `styled` (https://github.com/ksdme/org-status/blob/ea889698826230492e8e1fb44ccac7804ea6fe44/org_status/org_status.py#L104) to outside the for loop and instead print styled text (should probably be in red).

Reply:
You can use `f` string if you want to, this is a Python3 project.